### PR TITLE
Fix acquired semaphores num error when wait task finished

### PR DESCRIPTION
**Pull request by Linary**
Change-Id: I112261828af7267ba7463461153486af3ea97d4f

### CODE REVIEW

File: src/main/java/com/baidu/hugegraph/loader/HugeGraphLoader.java [hunk 1]
```diff
@@ -193,21 +196,21 @@ private void load(ElemType type) {
...
-            parseWatch.suspend();
+            parseTime.suspend();
             this.taskManager.submitBatch(struct, batch);
-            parseWatch.resume();
+            parseTime.resume();
         }
 
-        parseWatch.stop();
-        metrics.parseTime(parseWatch.getTime(TimeUnit.MILLISECONDS));
+        parseTime.stop();
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/loader/HugeGraphLoader.java]: move to line 205

File: src/main/java/com/baidu/hugegraph/loader/progress/LoadProgress.java [hunk 2]
```diff
@@ -65,6 +65,27 @@ public InputProgressMap get(ElemType type) {
...
 
+    public long totalLoaded(ElemType type) {
+        InputProgressMap lastLoadMap;
+        if (type.isVertex()) {
+            lastLoadMap = this.vertexProgress;
+        } else {
+            assert type.isEdge();
+            lastLoadMap = this.edgeProgress;
+        }
+
```
Comment (zhoney) [inline src/main/java/com/baidu/hugegraph/loader/progress/LoadProgress.java]:
```
InputProgressMap lastLoadMap = type.isVertex() ？ 
                               this.vertexProgress ：
                               this.edgeProgress;
```

File: src/main/java/com/baidu/hugegraph/loader/util/Printer.java [hunk 3]
```diff
@@ -41,13 +41,23 @@ public static void print(Object message) {
         System.out.println(message);
     }
 
-    public static void printElemType(ElemType type) {
+    public static void printRealTimeProgress(ElemType type, long count) {
+        String lastLoadMsg;
+        String thisLoadMsg;
```
Comment (zhoney) [inline src/main/java/com/baidu/hugegraph/loader/util/Printer.java]:
```
            lastLoadMsg = String.format(
                          "%s has been loaded last time : %s", type.string(), count);
            thisLoadMsg = String.format(
                          "%s has been loaded           : 0\b", type.string())
```

File: src/main/java/com/baidu/hugegraph/loader/HugeGraphLoader.java [hunk 4]
```diff
@@ -124,13 +124,22 @@ private void createSchema() {
         groovyExecutor.execute(script, client);
     }
 
+    private long lastLoadedCount(ElemType type) {
+        if (this.context.options().incrementalMode) {
+            return this.context.oldProgress().totalLoaded(type);
+        } else {
+            return 0;
```
Comment (javeme) [inline src/main/java/com/baidu/hugegraph/loader/HugeGraphLoader.java]: 0L

**Pull request merged**